bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/update/thing-update.xml

```diff
@@ -44,15 +44,6 @@
 			</add-channel>
 			<!-- channels to be updated -->
 		</instruction-set>
-		<instruction-set targetVersion="3">
-			<!-- channels to be removed -->
-			<!-- channels to be added -->
-			<!-- channels to be updated -->
-			<update-channel id="view" groupIds="image">
-				<type>mybmw:image-view-channel</type>
-				<label>Image Viewport</label>
-			</update-channel>
-		</instruction-set>
```

Comment:
I know I suggested to simply revert the previous commit, but that was unfortunately before checking its contents. Unfortunately it is not allowed to remove merged update instructions - see https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types

So you will have to bump thing type version to 4 and add new instructions for updating these channels again.

Reply:
when I created the PR I had the feeling that this might be required - and it seems logical. Done